x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/create_engine_flyout.tsx

```diff
@@ -127,44 +127,58 @@ export const CreateEngineFlyout = ({ onClose }: CreateEngineFlyoutProps) => {
         )}
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
-        <EuiSteps
-          steps={[
-            {
-              children: (
-                <IndicesSelectComboBox
-                  fullWidth
-                  isDisabled={formDisabled}
-                  onChange={onIndicesChange}
-                  selectedOptions={selectedIndices.map((index: string) => indexToOption(index))}
-                />
-              ),
-              status: indicesStatus,
-              title: i18n.translate(
-                'xpack.enterpriseSearch.content.engines.createEngine.selectIndices.title',
-                { defaultMessage: 'Select indices' }
-              ),
-            },
-            {
-              children: (
-                <EuiFieldText
-                  fullWidth
-                  disabled={formDisabled}
-                  placeholder={i18n.translate(
-                    'xpack.enterpriseSearch.content.engines.createEngine.nameEngine.placeholder',
-                    { defaultMessage: 'Search Application name' }
-                  )}
-                  value={engineName}
-                  onChange={(e) => setEngineName(e.target.value)}
-                />
-              ),
-              status: engineNameStatus,
-              title: i18n.translate(
-                'xpack.enterpriseSearch.content.engines.createEngine.nameEngine.title',
-                { defaultMessage: 'Name your Search Application' }
-              ),
-            },
-          ]}
-        />
+        <EuiFlexGroup direction="column">
+          <EuiFlexItem grow>
+            <EuiCallOut title="Technical Preview feature" color="warning" iconType="beaker">
+              <p>
```

Comment:
title and description are missing translations. [Guide](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md#defining-type-for-message) on how to do + some examples below of how i18n util is used.